### PR TITLE
override goodtimes

### PR DESCRIPTION
**Pull request by mahmoudimus**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -10,9 +10,14 @@ vg_vault_install_sftp: no
 vg_vault_rsyslog_host: localhost
 vg_vault_rsyslog_port: 514
 
+vg_vault_env: test
+vg_vault_home: /opt/vault/
 vg_vault_config_dir: /etc/verygood/
+vg_vault_config_env_dir: "{{ vg_vault_config_dir }}overrides/"
```
Comment (mahmoudimus) [inline defaults/main.yml]: probably want to call this the override dir.

File: templates/vault-http.supervisor.j2 [hunk 2]
```diff
@@ -1,8 +1,10 @@
+# {{ ansible_managed }}
 [program:vault-http]
-command=java -javaagent:{{ vg_vault_newrelic_home }}/newrelic.jar -Dnewrelic.environment={{ vg_vault_environment }} {{ vg_vault_http_jvm_options }} -jar {{ vg_vault_http_binary }} server {{ vg_vault_config_http }}
+command=java -javaagent:{{ vg_vault_newrelic_home }}/newrelic.jar -Dnewrelic.environment={{ vg_vault_environment }} -Dvault.http.env=vault.http.{{ vg_vault_env }}.yml {{ vg_vault_http_jvm_options }} -cp "{{ vg_vault_home }}*:{{ vg_vault_config_env_dir }}*:" com.verygood.payments.vault.VaultApplication server
```
Comment (mahmoudimus) [inline templates/vault-http.supervisor.j2]: definitely need to put back the yml file otherwise server won't start :-/

**Comment by mahmoudimus** [comment]
I think what was requested was that we get rid of the templated configuration completely and only set a template path to render a config file. We would also need to make sure that the production class path was available at the time.

**Comment by mjallday** [comment]
Closing in case we accidentally merge this into master.

**Pull request closed without merging**